README.md

```diff
@@ -1,254 +1,422 @@
+      > ```yaml
+      > - !policy
+      >   id: cf
+      >   body:
+      >     - !policy prod
+      >       owner: !group cf-admin-group
+      >       body:
+      >       - !layer cf/prod-apps
+      > 
+      >       - !host-factory
+      >         id: cf/prod-apps
+      >         layers: [ !layer cf/prod-apps ]
+      > ```
+      > 
+      > If you do not specify a `CONJUR_POLICY` (this is not recommended) in your Service Broker configuration and you are using > `CONJUR_VERSION` 4, then you need to add a Host Factory to the `root` Conjur policy by including:
+      > ```yaml
+      > - !layer apps
+      > 
+      > - !host-factory
+      >   id: apps
+      >   layers: [ !layer apps ]
+      > ```
+      > 
+
+    - `CONJUR_AUTHN_LOGIN`:
+      the identity of a Conjur Host (of the form `host/host-id`) with `create` and `update` privileges on `CONJUR_POLICY`. This account is used to add and remove Hosts from Conjur policy as apps are deployed to or removed from PCF.
+
+      If you are using Enterprise Conjur, you should add an annotation on the Service Broker Host in policy to indicate which platform the Service Broker is used on. The policy you load should similar to:
+      ```yaml
+      - !host
+        id: cf-service-broker
+        annotations:
+          platform: cloudfoundry
+      ```
+      You may elect to set `platform` to `cloudfoundry` or to `pivotalcloudfoundry`, for example. This annotation is used to set annotations on Hosts added by the Service Broker, so that they show in the Conjur UI with the appropriate platform logo.
+
+      > **NOTE:** The `CONJUR_AUTHN_LOGIN` value for the Host created in policy above is `host/cf-service-broker`.
+
+    - `CONJUR_AUTHN_API_KEY`:
+      the API Key of the Conjur Host whose identity you have provided in `CONJUR_AUTHN_LOGIN`.
 
-If you have reached this point, the Conjur Service Broker has been successfully
-deployed to your Cloud Foundry installation.
+    - `CONJUR_SSL_CERTIFICATE`: 
+      the PEM-encoded x509 CA certificate chain for Conjur. This is required if your Conjur installation uses SSL (e.g. Conjur Enterprise).
+
+        This value may be obtained by running the command:
+
+        ```sh-session
+        $ openssl s_client -showcerts -servername [CONJUR_DNS_NAME] \
+            -connect [CONJUR_DNS_NAME]:443 < /dev/null 2> /dev/null \
+            | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p'
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
+        ```
+
+    - `ENABLE_SPACE_IDENTITY`:
+      When set to the value `true`, the service broker provides applications with a Space-level `host` identity, rather than create a new `host` identity for the application in Conjur at bind time. This allows the broker to use a Conjur follower for application binding, rather than the Conjur master.
```

Comment:
```suggestion
      When set to the value `true`, the service broker provides applications with a Space-level `host` identity, rather than create a new `host` identity for each application in Conjur at bind time. This allows the broker to use a Conjur follower for application binding, rather than the Conjur master.
```